syz-ci/jobs.go

```diff
@@ -560,6 +560,8 @@ func (jp *JobProcessor) testPatch(job *Job, mgrcfg *mgrconfig.Config) error {
 		}
 	}
 	log.Logf(0, "job: testing...")
+	// TODO: now that there's no separate collide mode, we should probably consider transforming
```

Comment:
The reproducer should have detached flag enabled where required. Do you think it makes to "fuzz" it more? For 10 calls there 2^10 combinations of detached flags and lots of them will execute completely differently (e.g. some resource is not created before used). So by doing such fuzzing we can lower probability of triggering the original crash.